docs/source/blogs/release-0-3-0-blog/release-0-3-0-blog.rst

```diff
@@ -0,0 +1,192 @@
+
+
+.. code-block::
+
+  import glow
+  normalized_variants_df = glow.transform("normalize_variants", \
+    original_variants_df, \
+    reference_genome_path="/mnt/dbnucleus/dbgenomics/grch38/data/GRCh38_full_analysis_set_plus_decoy_hla.fa" \
+  )
+
+.. figure:: fignormdf.png
+   :align: center
+   :width: 800
+   :name: fignormdf
+
+   The normalized DataFrame ``normalized_variants_df``
+
+The output DataFrame of this improved transformer looks like :numref:`fignormdf`, where the ``start``, ``end``,  ``referenceAllele``, and ``alternateAlleles`` fields are updated by the normalized values and a ``normalizationStatus`` column is added to the DataFrame containing a ``changed`` subfield indicating whether the normalization changed the variant and an ``errorMessage`` subfield containing the error message in case of an error.
+
+
+The newly introduced ``replace_columns`` option can be used to add the normalization results as a new column to the DataFrame instead of replacing the original ``start``, ``end``,  ``referenceAllele``, and ``alternateAlleles`` fields. This can be done as follows:
+
+.. code-block::
+
+  import glow
+  normalized_variants_df = glow.transform("normalize_variants",\
+    original_variants_df, \
+    replace_columns="False", \
+    reference_genome_path="/mnt/dbnucleus/dbgenomics/grch38/data/GRCh38_full_analysis_set_plus_decoy_hla.fa" \
+  )
+
+.. figure:: fignormnorepdf.png
+   :align: center
+   :width: 800
+   :name: fignormnorepdf
+
+   The normalized DataFrame ``normalized_noreplace_variants_df`` with normalization results added as a new column
+
+The resulting DataFrame will be as shown in :numref:`fignormnorepdf`, where a ``normalizationResults`` column containing the normalized ``start``, ``end``,  ``referenceAllele``, ``alternateAlleles``, and ``normalizationStatus`` subfields is added to the DataFrame.
+
+
+We also note that since the multiallelic variant splitter is implemented as a separate transformer in this release (see below), the ``mode`` option of the ``normalize_variants`` transformer is deprecated. Refer to :ref:`variantnormalization` for more details on the ``normalize_variants`` transformer.
+
+
+``normalize_variant`` **function**: As mentioned, in this release, variant normalization can also be performed using the newly introduced ``normalize_variant`` SQL expression function as shown below:
+
+.. code-block::
+
+  from pyspark.sql.functions import expr
+  function_normalized_variants_df = original_variants_df.withColumn( \
+    "normalizationResult", \
+    expr("normalize_variant(contigName, start, end, referenceAllele, alternateAlleles, '/mnt/dbnucleus/dbgenomics/grch38/data/GRCh38_full_analysis_set_plus_decoy_hla.fa')") \
+  )
+
+As discussed in the previous section, this SQL expression function, like any other in Glow, now has Python and Scala APIs as well. Therefore, the same can be done in Python as follows:
```

Comment:
To simplify navigation, can you make any references to sections within the blog internal links?